Sources/PackageDescription/BuildSettings.swift

```diff
@@ -11,7 +11,7 @@
 //===----------------------------------------------------------------------===//
 
 /// The build configuration such as debug or release.
-public struct BuildConfiguration: Encodable {
+public struct BuildConfiguration: Codable {
```

Comment:
should these conformances move in PackageDescriptionSerialization.swift? the goal should be that the conformances are ~private to the code that does the specific serialization so we can have one to many formats

Reply:
Yep, that's one of the next steps. I'm just trying to avoid changing both ends of the serialization at the same time, so I am doing it in stages.